arbitrum-docs/for-devs/dev-tools-and-resources/web3-libraries-tools.mdx

```diff
@@ -10,16 +10,16 @@ import KnowMoreToolsBox from '../partials/_know-more-tools-box-partial.md';
 
 <KnowMoreToolsBox />
 
-In this page, we describe some of the popular libraries frameworks that exist for developing web3 decentralized applications. It’s worth noting that, because Arbitrum chains are EVM compatible, most of the tools that are used to develop on Ethereum, also provide support for Arbitrum.
+The following frameworks will help you build your decentralized apps (dApps):
 
 ### Ethers
 
-[Ethers.js](https://docs.ethers.org/) is a comprehensive and compact library designed for seamless interaction with the Ethereum, Arbitrum and other EVM-compatible blockchains and their ecosystems. With secure client-side private key management, compatibility with various nodes, and ENS name integration, it serves as a one-stop solution for dapp development needs. The library supports JSON wallet import/export, BIP 39 mnemonic phrases, and HD wallets, while remaining lightweight. Ethers.js also offers extensive documentation, full TypeScript readiness, and a generous collection of test cases, all under the open-source MIT License.
+[Ethers.js](https://docs.ethers.org/) is a comprehensive and compact library designed for seamless interaction with the Ethereum, Arbitrum and other EVM-compatible blockchains and their ecosystems. With secure client-side private key management, compatibility with various nodes, and ENS name integration, it serves as a one-stop solution for decentralized application development needs. The library supports JSON wallet import/export, BIP 39 mnemonic phrases, and HD wallets, while remaining lightweight. Ethers.js also offers extensive documentation, full TypeScript readiness, and a generous collection of test cases, all under the open-source MIT License.
 
 ### Web3.js
 
 [Web3.js](https://github.com/web3/web3.js/) is a versatile JavaScript library enabling developers to interact with Ethereum, Arbitrum and other EVM compatible nodes via HTTP, IPC, or WebSocket. It simplifies tasks like sending Ether, checking smart contract data, and creating contracts. As a JavaScript binding for the JSON RPC interface, web3.js is easily usable in popular web browsers, server-side Node.js applications, and Electron-based apps. Often used with the MetaMask browser extension, web3.js helps developers create decentralized applications and manage transactions directly from their preferred browser.
 
 ### Web3.py
 
-[Web3.py](https://github.com/ethereum/web3.py/) is a Python library designed for interacting with Ethereum, Arbitrum and other EVM compatible blockchains. It is widely used in decentralized apps (dapps) to manage transactions, work with smart contracts, and access block data. Originally derived from the Web3.js Javascript API, it has evolved to cater to the needs of Python developers. The library enables users to develop clients for reading and writing data on the blockchain, including smart contract interactions, making it a versatile tool for Ethereum-based applications.
```

Comment:
The [Style and convention guide](https://www.notion.so/arbitrum/Style-and-convention-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c) now contains terminology guidance that we can use to consistently refer to things like `JavaScript`, `dApps`, etc.